### ISSUE TITLE
Sub-elements of a div that does not have a div tag do not get indented

### ISSUE THREAD

**Issue by ajpauwels** [hlissner/emacs-pug-mode#28]
Example code, this works properly:
```
div#myExampleDiv
    p An example paragraph
```
When I write the `div`, then hit enter, and then hit tab, pug-mode correctly indents the `p` tag.

This does not work:
```
#myExampleDiv
    p An example paragraph
```
When the tag name is not provided, Pug assumes a `div` is desired. Pug-mode does not, however, detect that this is a valid `div` and does not properly indent the `p` tag inside the `div`.

This is not a big deal on new projects, as I think it's cleaner to specify the `div` tag explicitly, however when working in codebases where the tag-less style is used, it makes indenting a pain.

**Comment by hlissner** [comment]
Sorry for taking so long to get to this, but I'm trying to reproduce this and don't quite understand what the problem is. I've tried this myself and they both appear to indent correctly.

![pug](https://user-images.githubusercontent.com/510883/44621228-b5056c80-a8a2-11e8-8e59-937b937b2c34.gif)

Mind you, I've remapped `newline` to `newline-and-indent`, like so:

```elisp
(global-set-key [remap newline] #'newline-and-indent)
```

I've tried it with `newline` + <kbd>TAB</kbd>, but I get the same results. What am I looking for?

**Comment by ajpauwels** [comment]
You're right, the problem only happens one level of nesting deep. For example, try this:
```
#firstExampleDiv
	#secondExampleDiv
```

Now try indenting below `#secondExampleDiv` and it'll put the cursor at the same level as `#secondExampleDiv`, whereas I would like to have the option to indent into `#secondExampleDiv`
